agent_deploy/kubernetes/sysdig-agent-slim-daemonset-v2.yaml

```diff
@@ -77,6 +77,8 @@ spec:
           readOnly: true
       containers:
       - name: sysdig-agent
+        # WARNING: the agent-slim release currently must be run within
+        # a kubernetes cluster to properly function
```

Comment:
can we make this more specific? say, "NOTE: agent-slim depends on agent-kmodule so is supported only within Kubernetes".

Reply:
just fyi, agent-kmodule runs as an init container for agent-slim.

Reply:
is there something that a customer is expected to do with that information? that feels like an implementation detail

Reply:
ah i see what you are getting at. i'll change the comment